bbc_sale/models/stock_picking.py

```diff
@@ -31,13 +32,21 @@ def get_remarks(self):
             [purchase.remarks for purchase in purchases if purchase.remarks] +
             [sale.remarks for sale in sales])
 
+    @api.multi
+    def send_mail_outgoing_delivery(self):
+        """ Runs the server action of sending an email with the template
+        Magento | Send Email After Outgoing Delivery Is Shipped.
+        active_ids included in context because the server action has to
+        work in the Barcode Scanning Interface as well. """
+        template = self.env.ref(
+            'bbc_sale.action_send_email_delivery_shipped_magento')
+        self.env['ir.actions.server'].browse(template.id).with_context(
+            active_ids=self.ids).run()
```

Comment:
Something I missed earlier on: 
You retrieve the action and then browse the action's id. You can simply use

```
action = self.env.ref('bbc_sale.action_send_email_delivery_shipped_magento')
action.with_context(active_ids=self.ids).run()
```

Reply:
@StefanRijnhart You are right. Adjusted in my last commit.